package.json

```diff
@@ -103,7 +103,6 @@
     "react-scripts": "4.0.3",
     "redux-logger": "3.0.6",
     "redux-persist": "^6.0.0",
-    "rn-async-storage-flipper": "^0.0.10",
     "rn-flipper-async-storage-advanced": "^1.0.4",
```

Comment:
No need to upgrade this plugin?!

Reply:
No because just the front end part changed!

Reply:
So how do we fetch the front-end part? Just by upgrading the module in flipper?

Reply:
Yes. I do not know where to put it. I think I will just send a message to every flagship-app developers to update their front end in flipper, and new developers will automatically get the last one.